ohif-configurable/config-mode/src/toolbarButtons.js

```diff
@@ -384,7 +372,6 @@ const toolbarButtons = [
   // ************ New Layout Section
   
     // Measurement
-    isNewHP ? 
```

Comment:
Just get rid of the old menu as the new one has been around for a while - uses the new hanging protocol service.